astroquery/casda/tests/test_casda.py

```diff
@@ -10,28 +10,85 @@
 
 from astropy.coordinates import SkyCoord
 import astropy.units as u
-from astropy.table import Table
+from astropy.table import Table, Column
+from astropy.io.votable import parse
 
 from astroquery.casda import Casda
 
-DATA_FILES = {'CIRCLE': 'cone.xml', 'RANGE': 'box.xml'}
+try:
+    from unittest.mock import Mock, patch, PropertyMock, MagicMock
```

Comment:
As far as I know unittest.mock is available for all the python versions we support, so no need for the try/except. (But do correct me if I'm wrong, I never used this module)